seqr/views/apis/data_manager_api_tests.py

```diff
@@ -923,6 +926,14 @@ def _set_file_iter_stdout(rows):
                      f'{", ".join(sorted([col for col in header if col not in params["optional_headers"]]))}',
         })
 
+        mapping_body = {'mappingFile': {'uploadedFileId': 'map.tsv'}}
+        body.update(mapping_body)
+        mock_subprocess.side_effect = [mock_does_file_exist, mock_file_iter]
+        response = self.client.post(url, content_type='application/json', data=json.dumps(body))
+        self.assertEqual(response.status_code, 400)
+        self.assertDictEqual(response.json(), {'error': 'Must contain 2 columns: a'})
+
+        mock_load_uploaded_file.return_value = [['NA19675_D2', 'NA19675_1']]
```

Comment:
in the old model we stored the sample_id and if we tried to load data where the sample_id differed from the individual id we would look and see if there was already a sample model with that ID. We no longer store the unmapped sample_id, we just store the relationship to the individual, so now you have to explicitly provide a mapping from sampel_id to individual_id for all samples where they differ, instead of only the new ones